### PR TITLE
Create journal-of-the-botanical-research-institute-of-texas.csl

### PR DESCRIPTION
**Pull request by kshay-brit**
Citation style for the Journal of the Botanical Research Institute of Texas

### CODE REVIEW

File: journal-of-the-botanical-research-institute-of-texas.csl [hunk 1]
```diff
@@ -0,0 +1,639 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="display-and-sort" page-range-format="chicago">
+  <info>
+    <title>Journal of the Botanical Research Institute of Texas</title>
+    <title-short>JBRIT</title-short>
+    <id>https://csl.mendeley.com/styles/589017421/JBRIT</id>
```
Comment (POBrien333) [inline journal-of-the-botanical-research-institute-of-texas.csl:6]: Change to http://www.zotero.org/styles/journal-of-the-botanical-research-institute-of-texas

File: journal-of-the-botanical-research-institute-of-texas.csl [hunk 2]
```diff
@@ -0,0 +1,639 @@
...
+    <link href="http://www.zotero.org/styles/chicago-author-date" rel="self"/>
```
Comment (POBrien333) [inline journal-of-the-botanical-research-institute-of-texas.csl:7]: copy this line and make this one your template (rel="template") and then use this link as rel="self" http://www.zotero.org/styles/journal-of-the-botanical-research-institute-of-texas

File: journal-of-the-botanical-research-institute-of-texas.csl [hunk 3]
```diff
@@ -0,0 +1,639 @@
...
+    <link href="http://www.chicagomanualofstyle.org/tools_citationguide.html" rel="documentation"/>
```
Comment (POBrien333) [inline journal-of-the-botanical-research-institute-of-texas.csl:8]: Please link to the style guidelines.

File: journal-of-the-botanical-research-institute-of-texas.csl [hunk 4]
```diff
@@ -0,0 +1,639 @@
...
+      </if>
+    </choose>
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <name name-as-sort-order="first" and="text" sort-separator=", " delimiter=", " delimiter-precedes-last="always"/>
+      <label form="short" prefix=", "/>
+    </names>
+  </macro>
+  <macro name="translator">
```
Comment (POBrien333) [inline journal-of-the-botanical-research-institute-of-texas.csl:70]: Delete as it's not used.

File: journal-of-the-botanical-research-institute-of-texas.csl [hunk 5]
```diff
@@ -0,0 +1,639 @@
...
+          </names>
+          <names variable="editor translator" delimiter=", ">
+            <label form="verb" suffix=" "/>
+            <name and="text" delimiter=", "/>
+          </names>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="editor">
```
Comment (POBrien333) [inline journal-of-the-botanical-research-institute-of-texas.csl:64]: Delete macro as it's not used.

File: journal-of-the-botanical-research-institute-of-texas.csl [hunk 6]
```diff
@@ -0,0 +1,639 @@
...
+          <text macro="substitute-title"/>
+          <text macro="title"/>
+        </substitute>
+      </names>
+      <text macro="recipient"/>
+    </group>
+  </macro>
+  <macro name="contributors-short">
+    <names variable="author">
+      <name form="short" and="symbol" et-al-min="3" initialize-with=". "/>
```
Comment (POBrien333) [inline journal-of-the-botanical-research-institute-of-texas.csl:119]: if you set an et-al-min, then you'll also need to set et-al-min-first.

File: journal-of-the-botanical-research-institute-of-texas.csl [hunk 7]
```diff
@@ -0,0 +1,639 @@
...
+    <choose>
+      <if type="article-magazine article-newspaper review review-book" match="any">
+        <text macro="container-title"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="contributors">
+    <group delimiter=". ">
+      <names variable="author" font-variant="small-caps">
+        <name and="symbol" delimiter-precedes-last="always" et-al-min="10" initialize-with="." name-as-sort-order="first"/>
```
Comment (POBrien333) [inline journal-of-the-botanical-research-institute-of-texas.csl:104]: if you set an et-al-min, then you'll also need to set et-al-min-first.

**Comment by rmzelle** [comment]
@kshay-brit, thanks!

**Comment by kshay-brit** [comment]
Oh wow, thanks @rmzelle and @POBrien333! I'm sorry! I see I had a lot of errors in my submission. Thanks so much for revieiwng!!!

**Pull request merged**